ui/app/src/Utils/Views.elm

```diff
@@ -8,18 +8,12 @@ import Http exposing (Error(..))
 
 labelButton : Maybe msg -> String -> Html msg
 labelButton maybeMsg labelText =
-    let
-        label =
-            [ span [ class " badge badge-warning" ]
-                [ i [] [], text labelText ]
-            ]
-    in
-        case maybeMsg of
-            Nothing ->
-                span [ class "pl-2" ] label
+    case maybeMsg of
+        Nothing ->
+            Html.button [ class "btn btn-sm btn-outline-info ml-2 mb-2", disabled True ] [ text labelText ]
```

Comment:
Oh, sorry! I will put it back.  I could not check the silences because the endpoint that I'm using to test everything doesn't work with our new frontend